### PR TITLE
BAN-2660: Token listings

### PR DESCRIPTION
**Pull request by iamsphere**
## Issue link

https://linear.app/banx-gg/issue/BAN-2660/fe-banx-implement-listing-loans-feature-on-token-lending

## Vercel preview

https://banx-ui-git-feature-ban-2660-frakt.vercel.app/

### CODE REVIEW

File: src/pages/tokenLending/BorrowTokenPage/constants.ts [hunk 1]
```diff
@@ -2,6 +2,9 @@ import { LendingTokenType } from 'fbonds-core/lib/fbond-protocol/types'
 
 import { core } from '@banx/api/tokens'
 
+//? Default collateral mint (BANX token)
+export const DEFAULT_COLLATERAL_MINT = 'BANXbTpN8U2cU41FjPxe2Ti37PiT5cCxLUKDQZuJeMMR'
```
Comment (sablevsky) [inline src/pages/tokenLending/BorrowTokenPage/constants.ts:6]: тут тоже б взял из сдк

**Review by sablevsky — DISMISSED** [review]

File: src/pages/tokenLending/BorrowTokenPage/InstantBorrowContent/hooks/constants.ts [hunk 2]
```diff
@@ -1,3 +1 @@
-export const DEFAULT_COLLATERAL_MINT = 'BANXbTpN8U2cU41FjPxe2Ti37PiT5cCxLUKDQZuJeMMR'
```
[the remark this answers is not in the document]
Comment (iamsphere) [reply src/pages/tokenLending/BorrowTokenPage/InstantBorrowContent/hooks/constants.ts:1]: [Import BANX_TOKEN_MINT from sdk](https://github.com/frakt-solana/banx-ui/pull/1007/commits/611722544ac460c91afc51c70a1bc9f4aee5ce0f)

File: src/pages/tokenLending/BorrowTokenPage/constants.ts (hunk 1 above)
Comment (iamsphere) [reply src/pages/tokenLending/BorrowTokenPage/constants.ts]: [Import BANX_TOKEN_MINT from sdk](https://github.com/frakt-solana/banx-ui/pull/1007/commits/611722544ac460c91afc51c70a1bc9f4aee5ce0f)

File: src/pages/tokenLending/BorrowTokenPage/InstantBorrowContent/hooks/constants.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (iamsphere) [reply src/pages/tokenLending/BorrowTokenPage/InstantBorrowContent/hooks/constants.ts:1]: [Import BANX_TOKEN_MINT from sdk](https://github.com/frakt-solana/banx-ui/pull/1007/commits/611722544ac460c91afc51c70a1bc9f4aee5ce0f)

File: src/pages/tokenLending/BorrowTokenPage/constants.ts (hunk 1 above)
Comment (iamsphere) [reply src/pages/tokenLending/BorrowTokenPage/constants.ts]: [Import BANX_TOKEN_MINT from sdk](https://github.com/frakt-solana/banx-ui/pull/1007/commits/611722544ac460c91afc51c70a1bc9f4aee5ce0f)

**Pull request merged**